docs/reST/ref/image.rst

```diff
@@ -184,6 +184,17 @@ following formats.
 
    .. ## pygame.image.tostring ##
 
+.. function:: tobytes
+
+   | :sl:`transfer image to string buffer`
+   | :sg:`tobytes(Surface, format, flipped=False) -> bytes`
+
+   .. versionadded:: 2.1.3 added :func:`tobytes` as an alias for :func:`tostring`,
```

Comment:
Formatting here is a bit off from other places. I would change it so that the text after `versionadded` is the body/description of the function (and rewording it a little), and keeping just a `.. versionadded:: 2.1.3` at the end